src/Chapter2.hs

```diff
@@ -727,7 +750,8 @@ value of the element itself
 🕯 HINT: Use combination of 'map' and 'replicate'
 -}
 smartReplicate :: [Int] -> [Int]
-smartReplicate l = error "smartReplicate: Not implemented!"
+-- smartReplicate = concat . map (\x -> replicate x x)
+smartReplicate = ((\x -> replicate x x) =<<)
```

Comment:
Interesting usage of the `Monad instance in here 🔆 

`concatMap` is also one functions, so you could use it in here. It is usually a bit  more readable 🙂